### ISSUE TITLE
rdt-170M fine-tuning

### ISSUE THREAD

**Issue by yi-shui-han** [thu-ml/RoboticsDiffusionTransformer#58]
Hello, I have a question. When I was testing the fine-tuning of maniskill data, I selected `text_embed_PegInsertionSide-v1.pt` as the language embeddings, copied the `hdf5_maniskill_dataset.py` code to `hdf5_vla_dataset.py`, and executed the `finetune_maniskill.sh` script. However, after importing the file during training, the dimension of lang_embed was three, and after padding with `pad_sequence`, it became four-dimensional. Therefore, when indexing `L`, the dimension does not match. What is the reason?
![2024-12-24 21-46-50 的屏幕截图](https://github.com/user-attachments/assets/9fe00f57-9f00-402d-8c72-95c0e2396359)
![2024-12-24 21-54-36 的屏幕截图](https://github.com/user-attachments/assets/9f96374c-697f-4afb-baf0-3ebd516aa4a3)

**Comment by ethan-iai** [comment]
@LBG21

**Comment by LBG21** [comment]
Hi, the .pt files are used for inference and are not tested during fine-tuning (even though their values are the same). I believe they include an extra batch dimension. You can directly remove the 0th dimension (which should be 1) from `lang_embed` during fine-tuning.
